site/src/pages/WorkspaceSchedulePage/WorkspaceSchedulePage.tsx

```diff
@@ -184,13 +210,15 @@ export const WorkspaceSchedulePage: React.FC = () => {
         }}
       />
     )
-  } else if (scheduleState.matches("submitSuccess")) {
+  }
+
+  if (scheduleState.matches("submitSuccess")) {
     navigate(`/@${username}/${workspaceName}`)
     return <FullScreenLoader />
-  } else {
-    // Theoretically impossible - log and bail
-    console.error("WorkspaceSchedulePage: unknown state :: ", scheduleState)
-    navigate("/")
-    return null
   }
+
+  // Theoretically impossible - log and bail
```

Comment:
Potentially unpopular opinion up for debate: I think I'd rather the user run into an error state here so I know we've reached the 'theoretically impossible' state and we have a bug on their hands.

Reply:
I think we should still provide our users a way to get out of the error state. For example, in this case we are just redirecting to `/`. Maybe we can show them a generic error like "This should not have happened. Go to <a>workspaces</a>."

Reply:
Additionally, we can also show them the `Report an issue` link opening in new tab.